tests/unit/IOUUtilsTest.js

```diff
@@ -62,8 +62,8 @@ describe('IOUUtils', () => {
                 CONST.LOCALES.EN,
             );
 
-            // The starting point of all tests is the IOUReport containing a single non-pending transaction in USD
-            // All requests in the tests are assumed to be offline, unless isOnline is specified
+            // The starting point of all tests is the IOUReport containing a single pending transaction in USD
```

Comment:
```suggestion
            // The starting point of all tests is the IOUReport containing a single non-pending transaction in USD
```
This is still non-pending.